frontend-student/src/components/allcourses/Courses.jsx

```
@@ -1,37 +1,70 @@
+      setFilterType(type);
+      setFilterDirection('asc');
+    }
+  };
 
   const getThumbnail = (thumbnail) => {
     return thumbnail === 'str' ? noimage : thumbnail;
   };
 
-  // Animation variants
+  const truncateTitle = (title) => {
+    return title.length > 40 ? title.substring(0, 40) + '...' : title;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)